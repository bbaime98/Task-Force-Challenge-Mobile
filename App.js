import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import AppLoading from 'expo-app-loading';
import { useFonts  } from 'expo-font';
import {Provider} from 'react-redux'
import { NavigationContainer } from '@react-navigation/native';
// import 'react-native-gesture-handler';
import configureStore from './redux/configureStore'
import NeweTaskScreen from './screens/NewTaskScreen';
import StackNaviagtor from './navigation/stackNaviagtor';

const store = configureStore()
export default function App() {
  let [fontsLoaded] = useFonts({
    'bold': require('./assets/fonts/Montserrat-Bold.ttf'),
    'regular': require('./assets/fonts/Montserrat-Regular.ttf'),
    'semiBold': require('./assets/fonts/Montserrat-SemiBold.ttf'),
  });
  if (!fontsLoaded) {
    return <AppLoading />;
  }
  return (
    <Provider store={store}>
      <NavigationContainer>
      <StackNaviagtor />
    </NavigationContainer>
    </Provider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
