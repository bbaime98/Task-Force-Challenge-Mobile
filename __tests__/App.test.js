import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

jest.useFakeTimers()

describe('<App />', () => {
    it('should render snapshot', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
      });
  });
