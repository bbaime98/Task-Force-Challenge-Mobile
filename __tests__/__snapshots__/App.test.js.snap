// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `null`;

exports[`<App /> should render snapshot 1`] = `null`;

exports[`renders correctly 1`] = `null`;
